nvtabular/ops/list_slice.py

```diff
@@ -129,7 +129,7 @@ def transform(self, col_selector: ColumnSelector, df: DataFrameType) -> DataFram
 
     def _compute_dtype(self, col_schema, input_schema):
```

Comment:
Is this method  required to be overriden with this change? or could it be removed?

Reply:
It could be removed, but that breaks some of the tests even though the functionality works without it. (There are actually two places like this.)